app/assets/stylesheets/components/_navbar.scss

```diff
@@ -124,6 +124,25 @@
     }
   }
 
+  // .my-cards {
+  //   display: flex;
+  //   position: relative;
+  //   margin: 8px;
+  //   overflow: scroll;
+  // }
+  // .my-card {
+  //   height: 96px;
+  //   width: 80px;
+  //   margin: 64px 8px;
+  // }
+  // .my-card-image {
+  //   max-width: 72px;
+  //   max-height: 64px;
+  //   margin: 0;
+  // }
+  // .my-card-details {
+  //   display: none;
+  // }
```

Comment:
No it is like I said earlier in a previous pull request, This code was for the mobile show, but since it is destined to the navbar, I saved the commented code for the navbar, to be updated. I Simply moved it there for more clarity.